server/item/enchantment.go

```diff
@@ -39,6 +39,10 @@ type EnchantmentType interface {
 	Name() string
 	// MaxLevel returns the maximum level the enchantment should be able to have.
 	MaxLevel() int
+	// MinCost returns the minimum cost the enchantment may inhibit.
+	MinCost(level int) int
+	// MaxCost returns the maximum cost the enchantment may inhibit.
+	MaxCost(level int) int
```

Comment:
Would document this a bit more thoroughly, and maybe you could merge these into some sort of `CostRange()` method?

Reply:
Kind of an issue seeing as `MaxCost` is usually always `a.MinCost` + whatever factor.

Reply:
Updated.